bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -193,6 +197,28 @@ const getOpenChats = chatID => {
     return op;
   });
 
+  let currentClosedChats = Storage.getItem('closedChatList') || [];
+  let filteredChatList = [];
+
+  openChats.forEach((op) => {
+
+    // When user gets new messages, remove the chat from the session.
```

Comment:
The comment here could be reworded as `// When a new private chat message is received, ensure the conversation view is restored`